### DISCUSSION TITLE
New Release?

### DISCUSSION THREAD

**Discussion by devstein** [cyrilgdn/terraform-provider-postgresql#62] [General]
I would love to use the feature added in this [last PR](https://github.com/cyrilgdn/terraform-provider-postgresql/pull/59). Are there plans to do a release soon?

**Comment by cyrilgdn** [comment]
@devstein Yes, sorry for the delay, I'll release a new version this week (Wednesday at the latest).

**Reply by devstein** [reply]
Thanks @cyrilgdn !

**Reply by cyrilgdn** [reply]
@devstein A bit more late than expected but [v1.12.0 has just been released](https://registry.terraform.io/providers/cyrilgdn/postgresql/latest)

**Reply by devstein** [reply]
@cyrilgdn Appreciate it!
